store/tikv/tests/scan_test.go

```diff
@@ -105,10 +106,12 @@ func (s *testScanSuite) TestScan(c *C) {
 			// Because newScan return first item without calling scan.Next() just like go-hbase,
 			// for-loop count will decrease 1.
 			if i < rowNum-1 {
-				scan.Next()
+				err := scan.Next()
+				c.Assert(err, IsNil)
 			}
 		}
-		scan.Next()
+		// to fix: cannot check error, sometimes err not nil
```

Comment:
@disksing 
Could we handle this issue here?

Reply:
Not sure about the details. I think we can keep it for now.

Reply:
OK.
```suggestion
		// FIXME: Cannot check the error, sometimes `err` is not nil.
```